packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -42,5 +43,29 @@ export function addTooltipOptions<T extends OptionsWithTooltip>(
       settings: {
         options: sortOptions,
       },
+    })
+    .addSliderInput({
```

Comment:
slider input probablu won't allow a defult that is true "auto", right?

i think a number input here that is blank by default might be better here.

Reply:
just checked, if we remove `defaultValue` it will ignore this property at all and we gonna set default value inside TooltipPlugin2

Reply:
so, blank by default also works 🙂